client/tests/webdriver/specs/editPosition.spec.js

```diff
@@ -10,32 +10,29 @@ describe('Edit position page', () => {
     EditPosition.open()
   })
 
-  describe('When changing the position type from principal to advisor', () => {
-    it('Should reset the organization field', () => {
+  describe('When changing the position type from principal to advisor and saving', () => {
```

Comment:
You now put the 3 different suites in one suite, but the test description should also change than.
Maybe something like:
"When changing the position type"
and than in the it: "Should reset the organisation field and after a new organisation is selected and the form is submitted the updated position has the expected type."
Hmmm this is long, but this is because you now have quite some things you are testing in only one suite.

I would otherwise make the it more general but than add comments about the 3 things e are testing.